---
title: "Developing best practices for genotyping-by-sequencing analysis using linkage maps as benchmarks"
authors:
- Cristiane Hayumi Taniguti
- Lucas Mitsuo Taniguti
- Rodrigo Rampazo Amadeu
- Marcelo Mollinari
- Guilherme da Silva Pereira
- Oscar Riera-Lizarazu
- Jeekin Lau
- David Byrne
- Gabriel de Siqueira Gesteira
- admin
- Getulio Caixeta Ferreira
- Antonio Augusto Franco Garcia

date: "2022-11-25T00:00:00Z"
doi: "https://doi.org/10.1101/2022.11.24.517847"

# Schedule page publish date (NOT publication's date).
publishDate: "2022-11-25T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["3"]

# Publication name and optional abbreviated publication name.
publication: "*BioRxiv*"

abstract: <p align="justify">Genotyping-by-Sequencing (GBS) provides affordable methods for genotyping hundreds of individuals using millions of markers. However, this challenges bioinformatic procedures that must overcome possible artifacts such as the bias generated by PCR duplicates and sequencing errors. Genotyping errors lead to data that deviate from what is expected from regular meiosis. This, in turn, leads to difficulties in grouping and ordering markers resulting in inflated and incorrect linkage maps. Therefore, genotyping errors can be easily detected by linkage map quality evaluations. We developed and used the Reads2Map workflow to build linkage maps with simulated and empirical GBS data of diploid outcrossing populations. The workflows run GATK and freebayes for SNP calling and updog, polyRAD, and SuperMASSA for genotype calling, and OneMap and GUSMap to build linkage maps. Using simulated data, we observed which genotype call software fails in identifying common errors in GBS sequencing data and proposed specific filters to better handle them. We tested whether it is possible to overcome errors in a linkage map using genotype probabilities from each software or global error rates to estimate genetic distances with an updated version of OneMap. We also evaluated the impact of segregation distortion, contaminant samples, and haplotype-based multiallelic markers in the final linkage maps. The results showed a low impact of segregation distortion in the linkage map quality, improvements in ordering markers with haplotype-based multiallelic markers, and improved maps with expected size using reliable genotype probabilities or a global error rate of 5%. The pipelines results in each scenario changed according to the data set used, indicating that optimal pipelines and parameters are dataset-dependent and cannot be generalized to all GBS data sets. The Reads2Map workflow can reproduce the analysis in other GBS empirical data sets where users can select the pipeline and parameters adapted to their data context. The Reads2MapApp shiny app provides a graphical representation of the results to facilitate their interpretation.</p>

# Summary. An optional shortened abstract.
summary: Reads2Map improves GBS data analysis by addressing genotyping errors, optimizing linkage maps, and offering dataset-specific solutions with a user-friendly app.

tags:
- R Package
- Genetic
- Quantitative Genetic
- Reads2Map Tools
- Workflows

featured: true

links:
 - name: "Package Link"
   url: "https://github.com/Cristianetaniguti/Reads2MapTools"

url_pdf: ''
url_code: 'https://github.com/Cristianetaniguti/Reads2MapTools'
url_dataset: ''
url_poster: ''
url_project: 'https://github.com/Cristianetaniguti/Reads2MapTools'
url_slides: ''
url_source: ''
url_video: ''



# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
#image:
#  caption: 'Image credit: **Thiago de Paula Oliveira**'
#  focal_point: ""
#  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects:
- genetic-project

---
